.drone.yml

```
@@ -8,53 +8,30 @@ platform:
   arch: amd64
 
 steps:
-- name: my-build
-  pull: if-not-exists
-  image: docker:18.03
-  commands:
-  - docker build -t hocs_serenity_automation .
-  - docker run hocs_serenity_automation
-  environment:
-    DOCKER_HOST: tcp://172.17.0.1:2375
-  when:
-    branch:
-    - master
-    event:
-    - push
+- name: build project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    commands:
+      - ./gradlew build
+    when:
+      event:
+        - push
 
-- name: image_to_quay
-  pull: if-not-exists
-  image: quay.io/ukhomeofficedigital/drone-docker
+- name: build & push
+  image: plugins/docker
   settings:
     registry: quay.io
-    repo: quay.io/ukhomeofficedigital/hocs-serenity-automation
+    repo: quay.io/ukhomeofficedigital/hocs_serenity_automation
```

This should remain the old name. We're changing the tags so the Drone 1.0 tags (which start at 1) don't overwrite the old 0.8 ones -- but the repo can stay the same.
```suggestion
    repo: quay.io/ukhomeofficedigital/hocs-serenity-automation
```